### ISSUE TITLE
small world network as biomarker

### ISSUE THREAD

**Issue by edunnwe1** [openconnectome/Statistical-Connectomics-Class#224]
In this review: http://www.nature.com.ezproxy.welch.jhmi.edu/nrn/journal/v10/n3/full/nrn2575.html they discuss the prevalence of small-world networks in brain graphs and the variation between different age groups and diseased people. They suggest that perhaps small world network degradation could be a biomarker for disease. I thought this might be a bit tenuous because it relies so much on how you define the graph. I also have no idea how you could evaluate small-world network degradation statistically. What do you guys think? http://www.nature.com.ezproxy.welch.jhmi.edu/nrn/journal/v10/n3/full/nrn2575.html

**Comment by dlee138** [comment]
I have trouble understanding the papers definition of small-world networks (BOX3).  Are small world networks basically networks where every node is closely connected to other nodes not in terms of number of nodes they are connected to, but in terms of the short path (edges) between other nodes?  If so, I am also a bit skeptical about this idea as well because degree of small-world networkness seems difficult to quantify and doesn't seem like an all-or-none property.

**Comment by DSP137** [comment]
@dlee138 I think you are on the right track. I think a small-world network is one in which the longest path from one node to another is relatively small. (I'm thinking of the 6 degrees of separation thought.) 
@edunnwe1 For arguments sake, if you wanted to evaluate small-world network degradation, would there be a way to determine what the threshold for connectedness would need to be? Not sure how useful this would be in practice, but it's interesting to think about. I think we may end up with the same idea as in class when we discussed hubs - we new that a hub was a vertex with some degree property, but had a hard time deciding what the threshold should be to distinguish hub nodes versus non-hub nodes. In this case, what would we define as a threshold for what 'small' is with regards to longest path length.

**Comment by sgomezr** [comment]
Yes, I've always been skeptical about studies using small network topology because you don't know for sure if the way you're defining your graph is the best for the given situation. But there have been some papers focused mainly on explaining why the brain can be studied as a small world network. It does have some of the properties that are used for defining them, and some of the authors that have provided the framework (Bullmore and Sporns) are highly cited [3,4].

Yet, I've seen a lot of weird studies using small network topology on fMRI data or EEG recordings. The first one I came across with was  [1]. They were trying to relate small world network efficiency with the intelligence of the individual. First, they were using EEG, second, they were defining the network as a small world network by default and computing its efficiency (defined as the inverse of the average of the shortest path length), and third, they were assessing intelligence using RAPM (and intelligence test have always been a controversial). 

****These brings me to a question I'll post as an independent issue/comment/discussion: How do you compute a network from EEG recordings? fMRI is far more popular for constructing networks, then why use EEG? Can you really trust your networks constructed using EEG data? I'm just skeptical about them.... but don't answer me here, answer in the issue****

There's also [2] that uses  small network topology properties to compare the properties of graphs of awake and sleeping subjects.They obtained their data using EEG too, so that's the main reason I'm skeptical about them.... I may be wrong and those may be awesome studies that I'm failing to appreciate.

References:
1. Langer, Nicolas; Pedroni, Andreas; Functional brain network efficiency predicts intelligence.; Human brain mapping, 2012
2. Koenis, Maria M G; Romeijn, Nico, et al; Does sleep restore the topology of functional brain networks?; Human brain mapping; 2013
3. Bullmore E, Sporns O (2009): Complex brain networks: Graph the- oretical analysis of structural and functional systems. Nat Rev Neurosci
4. Bassett D, Bullmore E (2006): Small-world brain networks. Neuro- scientist

**Comment by yaxigeigei** [comment]
I was reading another paper, Structural and functional brain networks: from connections to cognition 2013 Science, where they use modules/submodules to express the similar idea and they also talked about rich-clubs. Although, I couldn't recall a specific point, my overall impression is similar to yours - the observations are volatile which is hard to be used as an standard measurement.

**Comment by whock** [comment]
So going back to the points raised about small-world networks (SWNs): how best to define SWNs? In @edunnwe1 's paper they define it as 

> Originally described in social networks 153 , the ‘small-world’ property combines high 
levels of local clustering among nodes of a network (to form families or cliques) and 
short paths that globally link all nodes of the network. This means that all nodes of a 
large system are linked through relatively few intermediate steps, despite the fact that 
most nodes maintain only a few direct connections — mostly within a clique of 
neighbours. 

so @dlee138 could you go into more detail about your skepticism? I agree this is network property exists on a continuum and its potential vagueness means we need to be careful with definitions.

**Comment by edunnwe1** [comment]
@DSP137  mightn't that be tricky since we want to measure degradation rather than a cut off? We might want something that tracked time as well, and was customizable to the individual subject's starting properties?

**Comment by DSP137** [comment]
@edunnwe1 Perhaps. I was thinking of degradation in terms of removal of vertices from a network, thus reducing network flow. In which case, if we remove certain vertices the communication lines from one (remaining) vertex to another (in terms of paths) may be drastically longer depending on what vertices were removed. It may be useful to find a way to track time as well. Are you thinking over a span of years? I think it would definitely be good to customize a bit based on a person's age and perhaps mental/physical state (if we can determine the correct associations to make there). How do you think we could go about doing that?

**Comment by mblohr** [comment]
Here is a description of the pros and cons of EEG vs fMRI and PET, from this website:
http://www.biomedresearches.com/root/pages/researches/epilepsy/eeg_fmri_and_pet.html

EEG vs fMRI and PET:

EEG has several strong points as a tool for exploring brain activity. EEG's can detect changes within a millisecond timeframe, excellent considering an action potential takes approximately 0.5-130 milliseconds to propagate across a single neuron, depending on the type of neuron. Other methods of looking at brain activity, such as PET and fMRI have time resolution between seconds and minutes. EEG measures the brain's electrical activity directly, while other methods record changes in blood flow (e.g., SPECT, fMRI) or metabolic activity (e.g., PET), which are indirect markers of brain electrical activity. EEG can be used simultaneously with fMRI so that high-temporal-resolution data can be recorded at the same time as high-spatial-resolution data, however, since the data derived from each occurs over a different time course, the data sets do not necessarily represent the exact same brain activity. There are technical difficulties associated with combining these two modalities, including the need to remove the MRI gradient artifact present during MRI acquisition and the ballistocardiographic artifact (resulting from the pulsatile motion of blood and tissue) from the EEG. Furthermore, currents can be induced in moving EEG electrode wires due to the magnetic field of the MRI.

**Comment by akim1** [comment]
Small world networks seem way too vague to be a biomarker for any type of disease. The level of a biomarker should give you a direct mapping to a particular disease. Also, not sure entirely sure why small world networks may be favored or disfavored for certain disease states.
